lib/internal/fs/cp/cp.js

```diff
@@ -306,7 +306,14 @@ function onDir(srcStat, destStat, src, dest, opts) {
 }
 
 async function mkDirAndCopy(srcMode, src, dest, opts) {
-  await mkdir(dest);
+  try {
+    await mkdir(dest);
+  } catch (error) {
+    // If the folder already exists, skip it.
+    if (error.code === 'EEXIST' && !opts.errorOnExist); else
+      throw error;
+  }
```

Comment:
When set to recursive, it will automatically ignore the EEXIST error. And the situation that needs to consider `force`.

```suggestion
  await mkdir(dest, {
   	recursive: opts.force || !opts.errorOnExist 
  });
```